### PR TITLE
chore(deps): Bump v6 branch to latest core alpha

### LINKED ISSUE #9411: React tokens -  consume core v6 alphas to enable penta development
Core is now publishing 6.0.0-alpha.x pacakges.

React has a v6 branch.

Deliverables for this task:
- consume the core alphas
- make sure the workspace builds
- make sure the global tokens living in https://github.com/patternfly/patternfly/tree/v6/src/patternfly/base/tokens are being turned into react-tokens

### PR DESCRIPTION
**Pull request by tlabaj**
**What**: Closes #9411

**Additional issues**: towards #9811, #9808,  and #9809  

- Bumped core alphas
-  Masthead:  Removed `backgroundColor` prop since light background is now the default with no option to change it.
- NotificationBadge:  `.pf-v5-c-notification-badge__count` no longer has a style associated with it so I updated the code with hardcoded placeholder string.
- PageSidebar: Removed Page sidebar `theme` prop since it only supports the default theme.
- PageSection: Removed `light`, `dark` and `darker variant since the are no longer supported.
- PageHeader (deprecated): removed `isSelected` prop since it is no longer supported in Page
- Fixed generate tokens to work with new naming 

Screenshots of tokens **BEFORE**:
Screenshot 2023-11-02 at 4 42 44 PM

image

Screenshots of tokens **AFTER**:
image

image

### CODE REVIEW

File: packages/react-docs/package.json [hunk 1]
```diff
@@ -23,7 +23,8 @@
     "test:a11y": "patternfly-a11y --config patternfly-a11y.config"
   },
   "dependencies": {
-    "@patternfly/patternfly": "5.1.0-prerelease.32",
+    "@patternfly/patternfly": "6.0.0-alpha.10"
+,
```
Comment (mfrances17) [inline packages/react-docs/package.json:27]: move , to prev line

File: packages/react-icons/package.json [hunk 2]
```diff
@@ -33,7 +33,8 @@
     "@fortawesome/free-brands-svg-icons": "^5.14.0",
     "@fortawesome/free-regular-svg-icons": "^5.14.0",
     "@fortawesome/free-solid-svg-icons": "^5.14.0",
-    "@patternfly/patternfly": "5.1.0-prerelease.32",
+    "@patternfly/patternfly": "6.0.0-alpha.10"
+,
```
Comment (mfrances17) [inline packages/react-icons/package.json:37]: move , to prev line :)

File: packages/react-styles/package.json [hunk 3]
```diff
@@ -19,7 +19,8 @@
     "clean": "rimraf dist css"
   },
   "devDependencies": {
-    "@patternfly/patternfly": "5.1.0-prerelease.32",
+    "@patternfly/patternfly": "6.0.0-alpha.10"
+,
```
Comment (mfrances17) [inline packages/react-styles/package.json:23]: same comment

File: packages/react-tokens/package.json [hunk 4]
```diff
@@ -29,7 +29,8 @@
     "clean": "rimraf dist"
   },
   "devDependencies": {
-    "@patternfly/patternfly": "5.1.0-prerelease.32",
+    "@patternfly/patternfly": "6.0.0-alpha.10"
+,
```
Comment (mfrances17) [inline packages/react-tokens/package.json:33]: same comment

**Review by mfrances17 — CHANGES_REQUESTED** [review]
just some nits, copy/paste error in multiple places :)

**Comment by nicolethoen** [comment]
it'd be good to double check that any global tokens in the core repo are getting translated appropriately into react-tokens, just like the other css variables.

**Review by srambach — COMMENTED** [review]
Notification badges don't look quite right - there's a weird outline and the hover is wrong. But these are also slightly still in flight along with some button work from @mcoker
Page is getting a scrollbar that isn't there in core. That might take a little more investigation.

**Review by nicolethoen — COMMENTED** [review]
This is really cool!

couple quick questions:

I see that we are defining the token for blue_100 for example as:
```
export const _pf_t_color_blue_100 = {
  "name": "--pf-t--color--blue--100",
  "value": "#d9e9ff",
  "var": "var(--pf-t--color--blue--100)"
};
export default _pf_t_color_blue_100;
```

Will that eventually be renamed and somehow replace the coexisting global variable?:

```
export const global_palette_blue_100 = {
  "name": "--pf-v5-global--palette--blue-100",
  "value": "#bee1f4",
  "var": "var(--pf-v5-global--palette--blue-100)"
};
export default global_palette_blue_100;
```
EDIT: this might be exactly what your TODO in the PR description is referring to - so never mind :)

Also, the individual token files look good, but in patternfly_variables.js I see quite a few tokens that have `value: undefined` such as:
```
"_pf_t_global_color_nonstatus_orangered_100": {
      "name": "--pf-t--global--color--nonstatus--orangered--100",
      "value": "undefined",
      "values": [
        "--pf-t--color--orangered--200",
        "undefined"
      ]
    },
```
Is that expected?

**Review by kmcfaul — APPROVED** [review]

**Comment by srambach** [comment]
I'm still seeing a difference in notification badge. I know there's an upcoming sprint to clean up notification badge but I don't know if there's a problem with what's in this PR or if it will be addressed. The workspace styles to make the dark background can be removed for sure.
image

Otherwise, it looks ok to me and I'm fine moving forward and circling back.

**Comment by tlabaj** [comment]
> I'm still seeing a difference in notification badge. I know there's an upcoming sprint to clean up notification badge but I don't know if there's a problem with what's in this PR or if it will be addressed. The workspace styles to make the dark background can be removed for sure. image
> 
> Otherwise, it looks ok to me and I'm fine moving forward and circling back.

@srambach This can be addressed by the issue for notification badge #9811 
The goal of the PR was to pull in alphas, make sure it builds and make sure Tokens are generated correctly.

**Review by srambach — APPROVED** [review]

**Review by mfrances17 — APPROVED** [review]
LGTM

**Pull request merged**